test/src/Game/Werewolf/Test/Engine.hs

```diff
@@ -205,18 +204,11 @@ prop_checkGameOverDoesNothingWhenAtLeastTwoAllegiancesAlive game =
             length (nub . map (_allegiance . _role) . filterAlive $ game' ^. players) > 1
             ==> not . isGameOver $ run_ checkGameOver game'
 
-prop_startGameStartsWithSeersTurnWhenSeersPresent :: [Player] -> Property
-prop_startGameStartsWithSeersTurnWhenSeersPresent players = and [
+prop_startGameStartsWithNightfallTurn :: [Player] -> Property
+prop_startGameStartsWithNightfallTurn players = and [
     any isSeer players,
     isRight . runExcept . runWriterT $ startGame "" players
```

Comment:
Can remove the `any isSeer players` here now and just have `isRight . runExcept . runWriterT $ startGame "" players ==> isNightfallTurn`

On that note, could we add a `isNightfallTurn` query? Ditto for `isDaybreakTurn` :).